build/checkstyle.xml

```diff
@@ -4,36 +4,35 @@
 
```

Comment:
Did a reformat on this file... there is really only 1 small change which was needed when upgrading to the latest checkstyle.